### ISSUE TITLE
Switching between 2 playlists with Custom Layouts has odd results

### ISSUE THREAD

**Issue by bjhiggins** [colloqi/pisignage-server#187]
TL;DR -  it appears that if I have more than 1 playlist using a custom layout scheduled - the player loads the custom layout for the first playlist *but does not* load the custom layout for the next playlist when it switches.  I feel this is a bug.

I have two Playlists (DHCP-Stats, MRTG-Internet) which I am trying to play on a single PiSignage RasberryPi.  Playlist DHCP-Stats uses the custom layout "custom_layout_3_over_1", while MRTG-Internet uses "custom_layout_2_rows_3_cols".  Both were created in the pisignage.com "Template Designer" interface - so I assume the templates are correct.

If "DHCP-Stats" is listed first as a scheduled playlist - the assets are displayed in the correct order (see Result Screenshots: 3-over-1-applied-to-DHCP-Stats.jpeg attached).  When the player switches to "MRTG-Internet", the RaspberryPi uses the layout from "DHCP-Stats" to display it's data (see Result Screenshots: 3-over-1-applied-to-MRTG-Internet.jpeg)

If "MRTG-Internet" is listed first as a scheduled playlist - the assets are displayed in the correct order (see Result Screenshots: 2x3-applied-to-MRTG-Internet.jpeg attached).  When the player switches to "DHCP-Stats", the RaspberryPi uses the layout from "MRTG-Internet" to display it's data (see Result Screenshots: 2x3-applied-to-DHCP-Stats.jpeg)

Result Screenshots:
[3-over-1-applied-to-DHCP-Stats.jpeg](https://user-images.githubusercontent.com/3186365/231234059-2dbabf88-1151-46b9-b747-71f93dd09926.jpeg) - Expected output
[3-over-1-applied-to-MRTG-Internet.jpeg](https://user-images.githubusercontent.com/3186365/231234072-f293bea6-9986-4f97-a5c7-b9af17ac08cb.jpeg) - Example of the bug
[2x3-applied-to-MRTG-Internet.jpeg](https://user-images.githubusercontent.com/3186365/231234048-2498a1de-7bd4-4b36-a574-cd3f0a73bacb.jpeg) - Expected output
[2x3-applied-to-DHCP-Stats.jpeg](https://user-images.githubusercontent.com/3186365/231234029-4fe9a759-d56b-4f13-9a54-96800dc937b6.jpeg) - Example of the bug

Custom Layouts:
[custom_layout_3_over_1.txt](https://github.com/colloqi/pisignage-server/files/11202861/custom_layout_3_over_1.txt)
[custom_layout_2_rows_3_cols.txt](https://github.com/colloqi/pisignage-server/files/11202864/custom_layout_2_rows_3_cols.txt)

![player-group](https://user-images.githubusercontent.com/3186365/231230759-aecef3ea-b3d4-414b-8548-8f78d7b06caf.png)
![MRTG-Internet](https://user-images.githubusercontent.com/3186365/231230776-ddabac81-5e20-49f8-a787-f97886734c94.png)
![DHCP-Stats](https://user-images.githubusercontent.com/3186365/231230789-6afb305c-cc4e-49c6-932f-3c3b176bca3b.png)

**Comment by bjhiggins** [comment]
Soooo... using two separate layouts on a single player is not a supported feature?  Is it somewhere on the roadmap?
